packages/tailwind/README.md

```diff
@@ -3,3 +3,159 @@
+
+```css
+border-radius: 7px 7px 7px 0
+```
+
+---
+
+## Recommended preset configuration
+
+Our recommendation is to use Kaizen's custom preset as the default for your project.
+
+```
+// tailwind.config.js
+const { Preset } = require("@kaizen/tailwind")
+
+module.exports = {
+  presets: [Preset],
+}
+```
+Our preset is built using Kaizen's [design-tokens package](https://github.com/cultureamp/kaizen-design-system/tree/main/packages/design-tokens). You can see the preset [here](https://github.com/cultureamp/kaizen-design-system/blob/main/packages/tailwind/src/tailwind-presets.ts).  
+This approach will overwrite the default Tailwind config.
+
+<br/>
+
+To extend upon the preset with values specific to your project, we recommend passing a theme object to `theme.extend`.
+<details>
+<summary>See a config example with implementation here</summary>
+
+```
+const { Preset } = require("@kaizen/tailwind")
+
+module.exports = {
+  presets: [Preset],
+  theme: {
+    extend: {
+      colors: {
+        myCoolNewColor: "#ffffff",
+        purple-100: "lime"
+      }
+    }
+  }
+}
+```
+
+Here, the Kaizen preset overwrites the default Tailwind preset, and the `extend` field adds in some new values.  
+Be careful though! While adding in a _new_ field to `colors` won't overwrite any existing ones, passing in an existing field _will_. In this example, `purple-100` has unfortunately been overwritten.
+
+`<p className="text-sky">...` ❌ Default TW config overwritten by `Preset`
+
+`<p className="text-myCoolNewColor">` ✅ New color added
+
+`<p className="text-purple-100">` ️❗️ Value from `Preset` overwritten to "Lime"
+
+`<p className="text-purple-200">` ✅ Value from `Preset` not overwritten
+</details>
+
+<br/>
+
+---
+
+## Suggested Editor Config
+
+The assumption is that you will be using VSCode as your main IDE. Your configuration may differ if you are using alternative like Webstorm, Atom, etc, but there is relatively decent support in their plugin ecosystems
+
+### Tailwindcss VSCode Intellisense plugin
+
+Basic config for the [VSCode plugin](https://marketplace.visualstudio.com/items?itemName=bradlc.vscode-tailwindcss) for Tailwind intellisense lives in your projects `settings.json`.
+
+To get intellisense use:
+
+
+```
+  "tailwindCSS.classAttributes": [
+    "class",
+    "className",
+    "ngClass",
+    "classNameOverride"
+  ]
+```
+
+These identify where tailwind will provide intellisense in your editor.
+
+---
+
+## Prerequisite knowledge
+
+This package is essentially a custom Tailwind preset. This package does not change how Tailwind works; it only changes the  compiled classes.  
+
+For example,
+Tailwind allows us to use atomic classes with psuedo selectors as inline styles, like so:
+```
+<p className="hover:w-1">...
+```
+`w-1` in this instance would compile to a css rule that sets `width: 0.25rem` to the element, if using Tailwind defaults.  
```

Comment:
Might be worth pointing out that this essentially translated to `4px` so it's easier to compare to the below.

Reply:
Thanks - ended up scrapping this section.